exercises/acronym/src/example/java/Acronym.java

```diff
@@ -2,7 +2,17 @@
 import java.util.regex.Pattern;
 
 public class Acronym {
-    public static String generate(String phrase){
+    private final String acronym;
+
+    public Acronym(String phrase) {
+        acronym = generateAcronym(phrase);
+    }
+
+    public String getAcronym() {
```

Comment:
🤔  `get()` vs. `getAcronym()`?

At this point, there is exactly one value in the object.  Cherry-picking [`ThreadLocal#get`](https://docs.oracle.com/javase/8/docs/api/java/lang/ThreadLocal.html#get--) as an example from the JDK.